File: Restaurant/Java/Dessert.java
package restaurant;

class Dessert extends Item{
	
	Dessert(){
	}
	
	Dessert(String name, double price){
		this.name = name;
		this.price = price;
	}
	
	String getName() {
		return this.name;
	}
	
	double getPrice() {
		return this.price;
	}
	
}

File: Restaurant/C++/Dessert.cpp
#include "Dessert.hpp"
#include <string>

Dessert::Dessert()
{}

Dessert::~Dessert()
{}

Dessert::Dessert(std::string name, double price)
{
	this->name = name;
	this->price = price;
}	

std::string Dessert::getName()
{
	return this->name;
}

double Dessert::getPrice()
{
	return this->price;
}

